apps/indexer/ponder.schema.ts

```diff
@@ -94,7 +96,7 @@ export const votingPowerHistory = onchainTable(
 export const delegation = onchainTable(
   "delegations",
   (drizzle) => ({
```

Comment:
Breaking schema change: The `votingPowerHistory` table references relations are using `deltaMod` field but the relations in `votingPowerHistoryRelations` are trying to match with `transfer.amount` and `delegation.delegatedValue`. This will cause foreign key constraint violations. The relations should match on `transactionHash` only, not on amount fields.